index.html

```diff
@@ -1613,8 +1614,9 @@ <h3>Algorithm</h3>
               generate an <a data-link-for="JsonLdErrorCode">invalid container mapping</a>
               error and abort processing if <a>processing mode</a> is `json-ld-1.0`.</li>
             <li>Set the <a>container mapping</a> of <var>definition</var> to
-              <var>container</var>.</li>
-            <li class="changed">If <a>container mapping</a> is `@type`:
+              <var>container</var>
+              <span class="changed">coercing to an <a>array</a>, if necessary</span>.</li>
+            <li class="changed">If the <a>container mapping</a> of <var>definition</var> includes `@type`:
```

Comment:
Should it be "includes `@type`", or "has a single item `@type`" ??
(the second is how I would have interpreted the old text... In any case, this shows that the previous wording was ambiguous and worth changing)

Reply:
The only possibilities are `"@type"`, `["@type"]`, and `["@type", "@set"]`, and this step should pertain to all of those. Any other combinations won't make it passed create term definition. So, "contains `@type`" should be correct.

Reply:
Ok then.